CONTRIBUTING.md

```diff
@@ -111,22 +115,49 @@ If you would like to suggest a new button idea, please create a [new discussion]
    git push origin new-branch
    ```
 4. To make sure your forked repository is up to date with this repository. Add this repository as the upstream repository by doing the following:
+
 ```
 git remote add upstream https://github.com/sButtons/sbuttons.git
 
 ```
+
 Then, to fetch from this repository:
+
 ```
 git fetch upstream
 git merge upstream/master master
 ```
+
 5. Go to your forked repository and press the “New pull request” button.
 6. Once the pull request is reviewed and approved, it will be merged.
 
 ### Note on abandoned pull requests
 
 If a pull request is left unchanged for a week after changes are requested and the issue was assigned to the person that sent the new pull request, they will be unassigned and someone else will be given the chance to work on it.
 
+### Adding Projects that make use of sButtons
+
+If you come accross any project that uses the sButtons npm package, please feel free to add them to this repository.
+
+Below are the steps you need to take when you're about to add any project.
+
+- Navigate through this codebase to /assets/js/projects.js
+- In `projects.js` you'd see an array containing objects of projects in the format below.
+
+```js
+{
+    name,
+    about,
+    author,
+    link_to_project,
+    image,
+    github,
+  },
+```
+
+- When you are adding a new project, all you need to do is create another object that would contain the details of the project.
+- Make sure all the object keys are having the required string value.
```

Comment:
Add that the person should add a screenshot or image for the project and where they should add it

Reply:
Oops! I forgot  that.

I'd make the corrections.